docs/tutorials/forum-application/2.intro-forumApp.md

```diff
@@ -0,0 +1,43 @@
+# Forum Application
+
+In this tutorial, we will build a Forum application using [ABCI 2.0](https://docs.cometbft.com/v1.0/spec/abci/).
+A Forum is an application where people can discuss topics by posting messages and responding to each other.
+
+We will demonstrate the use of `ABCI 2.0` methods like - `CheckTx`, `PrepareProposal`, `ProcessProposal`, `FinalizeBlock`,
+`ExtendVote` and `VerifyVoteExtension`
+
+## What does the application do?
+
+For the sake of simplicity, we will not be developing a fully functioning Forum Application. In our tutorial, we will focus
+on `sender` and `messages` and the censorship of ill-behaved forum users.
+
+The **Forum Application** accepts `sender` and `message` as input and checks if the `sender`
+has been banned or not. If the `sender` is banned, it cannot send a `message` on the Forum.
+
+If the `sender` is not banned, the `message` is checked for any curse words. If the `message` contains any curse words, the
+`sender` is banned. If not, the `message` is persisted in the application state.
+
+*Note: Curse words are defined in genesis, a copy of which is available with all nodes. To add a new curse word, the
+node must use `VoteExtension`*
+
+### Application Structure
+
+In order to follow this tutorial, you can refer to the repository that hosts the finished [ABCI 2.0 Forum Application](https://github.com/cometbft/abci-v2-forum-app).
```

Comment:
Ultimately, I think that code should be accompanying this tutorial

Reply:
Sounds good. I'll go ahead and add the code to the repo. It should make things easier to maintain, especially for future updates.